api/src/main/java/uk/gov/hmcts/fees2/register/api/controllers/mapper/FeeDtoMapper.java

```diff
@@ -153,7 +153,7 @@ public Fee toFee(LoaderRangedFeeDto request, String author) {
 
     public Fee2Dto toFeeDto(FeeVersion version) {
         Fee2Dto fee2Dto = toFeeDto(version.getFee());
-        fee2Dto.setCurrentVersion(toFeeVersionDto(version));
+        fee2Dto.setMatchingVersion(toFeeVersionDto(version));
```

Comment:
can you double check if it's a breaking change for other clients (payments etc) ?